types.ts

```diff
@@ -210,7 +212,25 @@ export enum PostSetupType {
 export type PostSetup = SetEndpoint;
 
 interface BaseAuthentication {
+  /**
+   * A function that is called when a user sets up a new account, that returns a name for
+   * the account to label that account in the UI. The users credentials are applied to any
+   * fetcher requests that this function makes. Typically, this function makes an API call
+   * to an API's "who am I" endpoint and returns a username.
+   *
+   * If omitted, or if the function returns an empty value, the account will be labeled
+   * with the creating user's Coda email address.
```

Comment:
In my experience in the UI it's labeled with the user's username, not email address.